src/observer.js

```diff
@@ -205,8 +211,10 @@ const reactiveMixin = {
           this.__$mobRenderEnd = Date.now();
         }
       });
-      if (exception)
+      if (exception) {
+        errorsReporter.emit(exception);
```

Comment:
> only additional value of this throw is that the error get's printed to the console
I think, that's a big deal, as `onError` is meant to be used only in specific scenarios (tracking errors with sentry etc.)